cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -318,7 +318,7 @@ The following commands are only executable against the `system` database:
 |===
 |Name                                                                                           | Category              | Description
 | <<administration-security-administration-database-access, ACCESS>>                            | Database              | Determines whether a user can access a specific database.
-| <<administration-security-administration-database-all, ALL DATABASE PRIVILEGES>>              | Database and schema   | Implies all privileges from the categories Database and Schema for a specific database.
+| <<administration-security-administration-database-all, ALL DATABASE PRIVILEGES>>              | Database and schema   | Implies most privileges from the categories Database and Schema for a specific database.
```

Comment:
Wouldn't it be better to list what privileges it actually grants? `most` is very fuzzy

Reply:
In other places I have mentioned which are left out but here it might be a good idea to update to say which are included yes. Will fix :)